package oop0318;

class Test{
	//1) try ~catch 직접이용한 예외처리
	public void view() {
		try {
			
		}catch(Exception e) {
			System.out.println(e);
		}
	}
	public void disp() {}
	
}

public class Test04_throws {

	public static void main(String[] args) {
		// 예외를 던진다
		//-> 메소드 호출 시 예외처리를 한꺼번에 모아서 처리한다

	}

}
